### PR TITLE
update model sensitivity description

### PR DESCRIPTION
**Pull request by huqiwen0313**

### CODE REVIEW

File: content/03.aim1.md [hunk 1]
```diff
@@ -67,6 +67,12 @@ NMI is an normalization of the Mutual Information (MI, measures the dependence o
 
 Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters.
 
+##### 4. Model sensitivity
+We found VAE model is sensitive to hyperparameter tuning, especially for deeper network.
```
Comment (cgreene) [inline content/03.aim1.md]: Probably should be:
> We found that the VAE method was sensitive to hyperparameter tuning, especially for deeper networks.

File: content/03.aim1.md [hunk 2]
```diff
@@ -67,6 +67,12 @@ NMI is an normalization of the Mutual Information (MI, measures the dependence o
...
+The performance of 3 - layer VAE changed dramatically after parameter tuning.
+Our results should discourage authors and reviewers from relying on empirical performance comparisons to evaluation model performance.
+For the detail work of model sensitivity, please see [@doi:10.1101/385534]
```
Comment (cgreene) [inline content/03.aim1.md]: Maybe:
> The details of these results are available in [@doi:10.1101/385534].

**Review by cgreene — COMMENTED** [review]
Just a couple things to change 👍

**Comment by huqiwen0313** [comment]
Thanks @cgreene ! I changed the description a little. If it is ok, I will merge this PR.

File: content/03.aim1.md [hunk 3]
```diff
@@ -67,6 +67,12 @@ NMI is an normalization of the Mutual Information (MI, measures the dependence o
 
 Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters.
 
+##### 4. Model sensitivity
+We found VAE model was sensitive to hyperparameter tuning, especially for deeper network.
```
Comment (cgreene) [inline content/03.aim1.md]: network -> networks

File: content/03.aim1.md [hunk 4]
```diff
@@ -67,6 +67,12 @@ NMI is an normalization of the Mutual Information (MI, measures the dependence o
...
+The performance of 3 - layer VAE changed dramatically after parameter tuning.
```
Comment (cgreene) [inline content/03.aim1.md]: `3-layer` without the space

File: content/03.aim1.md [hunk 5]
```diff
@@ -67,6 +67,12 @@ NMI is an normalization of the Mutual Information (MI, measures the dependence o
...
+Our results should discourage authors and reviewers from relying on empirical performance comparisons to evaluation model performance.
```
Comment (cgreene) [inline content/03.aim1.md]: `evaluation` -> `evaluate`

also, maybe `from solely relying`

File: content/03.aim1.md [hunk 6]
```diff
@@ -67,6 +67,12 @@ NMI is an normalization of the Mutual Information (MI, measures the dependence o
 
 Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters.
 
+##### 4. Model sensitivity
```
Comment (cgreene) [inline content/03.aim1.md:70]: Do you need a blank line after the header? I think markdown might expect it.

**Comment by cgreene** [comment]
Saw a few more things on another read through. Once you get them feel free to merge.

**Pull request merged**